Source/Core/Common/GL/GLInterface/EGLDRM.h

```diff
@@ -0,0 +1,55 @@
+// Copyright 2008 Dolphin Emulator Project
+// Licensed under GPLv2+
+// Refer to the license.txt file included.
+
+#pragma once
+
+#include <EGL/egl.h>
+#include <EGL/eglext.h>
+#include <string>
+#include <vector>
+
+#include "Common/GL/GLContext.h"
+
+typedef struct
```

Comment:
In C++ this can formatted more concisely like so (with naming altered to follow our naming conventions):

```cpp
struct EGLContextData
{
   EGLContext ctx;
   EGLSurface surf;
   EGLDisplay dpy;
   EGLConfig config;
   int interval;

   unsigned major;
   unsigned minor;
};
```

Reply:
Fixed in c55dc590e461bc7e9d647ad0cb8e0d7ff50044ae